tests/validation/test_evaluators.py

```diff
@@ -277,6 +277,19 @@ def test_spearman_evaluator():
     assert result['spearman_evaluator__target'] == 1.0
 
 
+def test_ndcg_evaluator():
```

Comment:
We are missing just two tests here:
- When user provides `k` different from the size of the rank
- When use choose to don't use `exponential_gain`

Tip: You can use parametrize to use this same test and just pass different arguments, just like [here](https://github.com/nubank/fklearn/blob/fddab4d207675ee662fc5cb2091abdf6503d2990/tests/training/test_pipeline.py#L105)

Reply:
Hi, Caique!

I agree, thanks for the tip, it's was helpful!

I already made the changes.